a3p-integration/proposals/a:vaults-auctions/saveAuctionInstance.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+import { writeFile } from 'fs/promises';
+import { getAuctionInstance } from './agd-tools.js';
+
+const { env } = process;
+
+const oldAuctionInstance = await getAuctionInstance();
+console.log('old auction instance ', oldAuctionInstance, env.HOME);
+
+await writeFile(
+  `${env.HOME}/.agoric/previousInstance.json`,
```

Comment:
this file is part of the permanent history a3p. please give it a more informative name for someone who comes across it without this context. or delete it after it's read.

also in general I think f a3p is writing to this directory it should be under a sub-path